Integrations/AlienVault_USM_Anywhere/AlienVault_USM_Anywhere.py

```diff
@@ -422,6 +422,12 @@ def fetch_incidents():
     if incidents:
         #  updating according to latest incident
         time_str = str(incidents[-1].get('occurred'))
+
+        # add one second to last incident occurred time to avoid duplications
+        occurred = datetime.strptime(time_str, '%Y-%m-%dT%H:%M:%S.000Z')
```

Comment:
will `time_str` always have zero milliseconds? if not, truncate the string and parse only the beginning